docs/book/basics/simple_controller_manager.md

```diff
@@ -46,3 +46,34 @@ func main() {
   log.Fatal(mgr.Start(signals.SetupSignalHandler()))}
 ```
 {% endmethod %}
+
+{% method %}
+## Leader Election
```

Comment:
I would suggest moving this under `beyond the basics/controllers/` right after `Using finalizers`